modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/IgniteThrottlingUnitTest.java

```diff
@@ -76,8 +77,11 @@
 
         IgniteConfiguration cfg = new IgniteConfiguration().setMetricExporterSpi(new NoopMetricExporterSpi());
 
+        PerformanceStatisticsProcessor perfStatProc = mock(PerformanceStatisticsProcessor.class);
+
         DataRegionMetricsImpl metrics = new DataRegionMetricsImpl(new DataRegionConfiguration(),
-            new GridMetricManager(new GridTestKernalContext(new GridTestLog4jLogger(), cfg)), NO_OP_METRICS);
+            new GridMetricManager(new GridTestKernalContext(new GridTestLog4jLogger(), cfg)), perfStatProc,
```

Comment:
Can we use `GridTestKernalContext` here for `PerformanceStatisticsProcessor` also instead of mock.

Reply:
done